pkgs/development/python-modules/py-dmidecode/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, dmidecode }:
+
+buildPythonPackage rec {
+  pname = "py-dmidecode";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1bv1vmhj8h520kj6slwpz16xfmgp117yjjkfyihkl5ix6mn5zkpa";
+  };
+
+  propagatedBuildInputs = [ dmidecode ];
+
```

The project has no tests (-> `Ran 0 tests in 0.000s`). 

```suggestion
  propagatedBuildInputs = [ dmidecode ];

  # Project has no tests
  doCheck = false;
  pythonImportsCheck = [ "dmidecode" ];

```

---

Fixed!